### PR TITLE
loginctl: add user logout example

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/loginctl.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - Execute a `loginctl` operation on a remote host:
 
 `loginctl list-users -H {{hostname}}`
+
+- Log a user out on all of its sessions:
+
+`loginctl terminate-user {{username}}`
```
Comment (cyqsimon) [inline pages/linux/loginctl.md:28]: Should we have `sudo` here?

```suggestion
`sudo loginctl terminate-user {{username}}`
```

The reason I'm not sure is, despite the fact that this command requires privilege escalation, it actually works without `sudo` too by authenticating using polkit.

Current guidelines really doesn't make it clear whether `sudo` should be included in such a case. I think it depends on how we interpret its purpose: is it here to

1. make the command work, or
2. indicate privilege escalation?

Personally I favour the second interpretation so I prefer adding `sudo`. I'd like to hear others' opinions.

**Review by cyqsimon — APPROVED** [review]

File: pages/linux/loginctl.md (hunk 1 above)
Comment (Managor) [reply pages/linux/loginctl.md:28]: Loginctl without sudo applies only to the current user as in you can log yourself out of all of your instances. If you do not povide sudo and attempt to use it on someone else, the tool will ask for sudo password anyways.

Comment (gutjuri) [reply pages/linux/loginctl.md:28]: > Thus I don't see that adding sudo is necessary.

+1

Comment (Managor) [reply pages/linux/loginctl.md:28]: Systemctl acts the same way and its page doesn't use sudo.

File: pages/linux/loginctl.md [hunk 2]
```diff
@@ -22,3 +22,7 @@
 - Execute a `loginctl` operation on a remote host:
 
 `loginctl list-users -H {{hostname}}`
+
+- Log a user out on all of its sessions:
```
Comment (gutjuri) [inline pages/linux/loginctl.md:26]:
```suggestion
- Log a user out on all of their sessions:
```

**Review by gutjuri — APPROVED** [review]

**Review by karthik-script — APPROVED** [review]

**Pull request merged**